src/main/java/com/google/codeu/servlets/BlobstoreRestaurantUploadServlet.java

```diff
@@ -1,4 +1,4 @@
-package io.happycoding.servlets;
+package com.google.codeu.servlets;
```

Comment:
This line breaks image uploading for me. When I try to upload an image to the URL, the message only shows a little box with a question mark.